hack/testdata/filter/pod-apply-selector.yaml

```diff
@@ -8,4 +8,4 @@ metadata:
 spec:
   containers:
   - name: kubernetes-pause
-    image: k8s.gcr.io/pause:2.0
+    image: k8s.gcr.io/pause:3.4.1
```

Comment:
i wonder if these are used even. 2.0 was quite some time ago.

i'd recommend leaving all hack/testdata changes out of the PR, but let's see comments from others first.

also these files should be ideally templated instead.

Reply:
OK. 

I think this will make it easier in the future when we need more integration or e2e tests on windows.